app/src/common/shared/org/mozilla/vrbrowser/input/DeviceType.java

```diff
@@ -0,0 +1,24 @@
+package org.mozilla.vrbrowser.input;
+
+import android.util.Log;
+
+public class DeviceType {
+    // These values need to match those in Device.h
+    public static final int Unknown = 0;
+    public static final int OculusGo = 1;
+    public static final int OculusQuest = 2;
```

Comment:
we should probably remember to add the others, but this is fine for now

Reply:
I only wanted to add what we needed.